.github/workflows/ci_release.yml

```diff
@@ -111,19 +123,35 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GPG_FINGERPRINT: ${{ steps.import_gpg.outputs.fingerprint }}
+          GORELEASER_CURRENT_TAG: $LATEST_TAG
 
   upload-docs:
-    needs: goreleaser
-    if: github.event_name == 'release'
+    needs: [setup]
+    if: |
+      github.event_name == 'release' ||
+      github.event_name == 'workflow_dispatch'
     runs-on: ubuntu-latest
     permissions:
       contents: write
     steps:
       - uses: actions/checkout@v4
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set LATEST_TAG for release
+        if: github.event_name == 'release'
+        run: echo "LATEST_TAG=${{ github.event.release.tag_name }}" >> $GITHUB_ENV
+
+      - name: Set LATEST_TAG for workflow_dispatch
+        if: github.event_name == 'workflow_dispatch'
+        run: echo "LATEST_TAG=${{ github.event.inputs.run-on-tag }}" >> $GITHUB_ENV
+
+      - name: Checkout a given tag
+        if: github.event_name == 'workflow_dispatch'
+        run: git checkout ${{ inputs.run-on-tag }}
```

Comment:
Should checkout to LATEST_TAG instead?

Reply:
Doesn't matter 'cause it's the same as in `workflow_dispatch`.